assets/css/admin/style.scss

```diff
@@ -669,24 +669,30 @@ table.edd-order-overview-summary--refund {
 }
 
 /** Totals */
-.edd-order-overview-summary__totals .column-primary {
+.edd-order-overview-summary__subtotal .column-primary,
```

Comment:
I'm wondering if it would be tidier to make a common class which could be used for the subtotal/adjustments/total `tbody` so that we could have just one rule for these.

Reply:
Looking again I think I like having these separate and we can adjust some of the repetition when more properly converted to `.scss.`